sdk/storage/Azure.Storage.Blobs/src/BlobServiceClient.cs

```diff
@@ -1654,10 +1654,11 @@ internal BlobServiceClient(Uri serviceUri, HttpPipelinePolicy authentication, Bl
         /// Find Blobs by Tags</see>.
         /// </summary>
         /// <param name="tagFilterSqlExpression">
-        /// The where parameter enables the caller to query blobs whose tags match a given expression.
-        /// The given expression must evaluate to true for a blob to be returned in the results.
-        /// The[OData - ABNF] filter syntax rule defines the formal grammar for the value of the where query parameter;
-        /// however, only a subset of the OData filter syntax is supported in the Blob service.
+        /// The where URI parameter finds blobs in the storage account whose tags match a given expression.
```

Comment:
Is this supposed to be "The where URI parameter.." ?

Reply:
May we should change it to "The where parameter..."  I copy + pasted it from the rest docs.